release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3279,6 +3279,11 @@ data:
 +
 (link:https://issues.redhat.com//browse/OCPBUGS-35316[*OCPBUGS-35316*])
 
+* Low-latency applications that rely on high-resolution timers to wake up their threads might experience higher wake up latencies than expected. 
+Although the expected wake up latency is under 20μs, latencies exceeding this can occasionally be seen when running the `cyclictest` tool for long durations (24 hours or more). 
```

Comment:
```suggestion
Although the expected wake up latency is under 20μs, latencies exceeding time/value this can occasionally be seen when running the `cyclictest` tool for long durations (24 hours or more). 
```